config/application.yml

```diff
@@ -12,4 +12,4 @@ YOUTUBE_API_KEY: AIzaSyDyY__HGUskUuI_HVQg2wOVpJZZBiEoDKU
 # production:
 #   stripe_api_key: sk_live_EeHnL644i6zo4Iyq4v1KdV9H
 #   stripe_publishable_key: pk_live_9lcthxpSIHbGwmdO941O1XVU
-GH_API_KEY: token 49475bf778cbfec543a25d65e0bf5ed5f54ed18e
\ No newline at end of file
+# GH_API_KEY: token 49475bf778cbfec543a25d65e0bf5ed5f54ed18e
```

Comment:
Opps I just nuked youre token.. My bad thought I pulled it out..

Reply:
lol damn it jack